index.html

```diff
@@ -20,9 +20,15 @@ <h2>You Know the Rules</h2>
     </section>
     <section class="weapon-selection">
         <h3>Select Your Weapon</h3>
-            <label for="user-play">Rock <input type="radio" name="user-play" value="rock"></label>
-            <label for="user-play">Paper <input type="radio" name="user-play" value="paper"></label>
-            <label for="user-play">Scissors <input type="radio" name="user-play" value="scissors"></label>
+            <label>
+                <input type="radio" name="user-play" value="rock" class="hidden" checked><img src="https://vignette.wikia.nocookie.net/yandere-simulator-fanon/images/4/4e/Johnson.jpg/revision/latest?cb=20160625153352" height="150px" alt="rock" class="weapon">
```

Comment:
You can always save your images into an assets folder in VScode and directly path them in your code. That way your images won't break in case the original link source does.